src/multivariate/mvnormal.jl

```diff
@@ -372,7 +372,7 @@ struct MvNormalStats <: SufficientStats
     tw::Float64         # total sample weight
 end
 
-function suffstats(D::Type{MvNormal}, x::AbstractMatrix{Float64})
+function suffstats(::Type{MvNormal}, x::AbstractMatrix{Float64})
```

Comment:
Maybe move unrelated changes to a separate PR?

Reply:
there were all relatively minor things (unused variables)